src/gui/widgets/multiline_text.cpp

```diff
@@ -406,7 +435,27 @@ void multiline_text::signal_handler_left_button_down(const event::ui_event event
 	get_window()->keyboard_capture(this);
 	get_window()->mouse_capture();
 
-	handle_mouse_selection(get_mouse_position(), true);
+	point mouse_pos = get_mouse_position();
+
+	if (get_link_aware()) {
+		if (desktop::open_object_is_supported()) {
+			std::string link = get_label_link(mouse_pos - get_origin());
+			if (!link.empty()) {
+				DBG_GUI_E << "Clicked Link:\"" << link << "\"";
+
+				const int res = show_message(_("Open link?"), link, dialogs::message::yes_no_buttons);
+				if(res == gui2::retval::OK) {
+					desktop::open_object(link);
+				} else {
+					handle_mouse_selection(get_mouse_position(), true);
+				}
+			}
+		} else {
+			show_message("", _("Opening links is not supported, contact your packager"), dialogs::message::auto_close);
```

Comment:
I suggest copying it instead in this case (and mentioning that in the message).

Reply:
Updated.